framework/config/config.py

```diff
@@ -88,7 +85,7 @@ def initialize_attributes(self):
     def LoadFrameworkConfigFromFile(self, config_path):
         """Load the configuration from into a global dictionary."""
```

Comment:
https://github.com/hTrap/owtf/blob/pep8-fixes/framework/config/config.py#L195
Not needed here, just `logging.info(e.parameter)` is fine.